### DISCUSSION TITLE
More arranger playback options

### DISCUSSION THREAD

**Discussion by phpguru** [SynthstromAudible/DelugeFirmware#2603] [Ideas]
Maybe there is another way to do this that I haven't learned yet but in arranger mode, it would be convenient to be able to hold down a pad (eg on bar 17) and press play - the arrangement would start playing on bar 17. This would save time (of playing from the beginning to bar 17 each time) while building the arrangement.

Another thing I would like to be able to do in arranger is set the _Arranger UI Playback Mode_ (I made that up) to "fixed" - in this mode the white playback stripe would remain on the left-most column 1 of the grid (maybe half-lit) and the arrangement clips would stream to the left as the arrangement plays. I find myself turning the <> knob with the beat to keep it on column 1. The default Arranger UI Playback Mode is like "fluid," "normal," or "default," where the <> and ^v knobs allow scrolling around. Fixed mode is like how Mario is always in the center of the screen and the world moves in the background.

I'm not sure how to achieve this in the UI - maybe a long press on the track puts you into the clip - is it possible to mute synth notes or kit sounds in arranger mode? Example: Bar 1-2, the kit plays only the kick. Bar 3-4 I bring in the hats and shakers, but it's from the same clip, just unmuting parts every 2 bars for 16 bars.

**Comment by ok-reza** [comment]
1) something like this is being developed currently

2) press cross-screen, this will make the arrangement scroll and the white cursor stay in the same place (you can even turn <> to adjust which column the white cursor stays on.

**Reply by xophos-commits** [reply]
will similar behavior be available for clip view in the future?
